scss/main.scss

```diff
@@ -179,3 +166,45 @@ $grey: #C4C4C4;
     overflow: hidden;
     transition: max-height 0.1s ease-out;
  }
+
+ .sponsors-list {
+   overflow: hidden;
+ }
+
+.sponsor-img {
+    width: 100px;
+    height: 100px;
+    z-index: 1;
+    margin: 0px 50px;
+}
+
+.contact{
+    background-color: $light-grey;
+    text-align: center;
+}
+
+.contact-content a:link {
+    color: #000;
+}
+
+//contact us circles
+.blue-circle {
```

Comment:
oop can u run the 'npm run web' again? it looks like some of ur changes didnt go to the .css file. everything else looks good tho!!